pkg/iac/terraform/state/enumerator/state_enumerator.go

```diff
@@ -7,6 +7,7 @@ import (
 )
 
 type StateEnumerator interface {
+	Source() string
```

Comment:
I think Source there is a bit confusing with recently introduced resource sources, don't you think ? We could name it Path()

Reply:
yeah you're right